src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -947,7 +948,7 @@ public void onFileChange(File file) {
             // Parse hostname, http, https ports for integration tests to use
             parseHostNameAndPorts(serverTask, messagesLogFile);
         } catch (IOException e) {
-            throw new PluginExecutionException("An error occurred while starting the server: " + e.getMessage(), e);
+            throw new PluginExecutionException("An error occurred while starting the server: " + e.getMessage());
```

Comment:
I don't think we should do these changes. There is no security risk here because this is all open source. We need this level of detail to support our customers when they report issues. We can document this reasoning in the spreadsheet for all of this category of issues.